microprofile/microprofile-on-micro/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.examples</groupId>
+        <artifactId>microprofile</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>MicroProfileOnMicro</artifactId>
+    <packaging>war</packaging>
+
+    <name>MicroProfileOnMicro</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+        <version.jakartaee>8.0.0</version.jakartaee>
+        <version.payara>5.193</version.payara>
```

Comment:
these should be defined in a parent pom so they only need updating once

Reply:
Hi @Pandrex247, can you please address the feedback above?

Reply:
Should they though?
This is what this example was written against - other examples may not be using Metrics 2.2 or Jakarta EE 8 for example

Reply:
I think they should otherwise Payara Examples becomes one great mess downloading many different versions of Micro and other artefacts. Examples should be updated regularly to the latest specs and runtimes.

Reply:
OK I'll need to have a look at this again since there are breaking changes in the API between 2.2 and 3